var/spack/repos/builtin/packages/ima-evm-utils/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ImaEvmUtils(AutotoolsPackage):
+    """IMA/EVM control utilities."""
+
+    homepage = "http://linux-ima.sourceforge.net"
+    url      = "http://sourceforge.net/projects/linux-ima/files/ima-evm-utils/ima-evm-utils-1.2.1.tar.gz"
+
+    version('1.3.1', sha256='5304271f31a3601a2af5984942d9bd6c7532597c5a97250c9a4524074fc39925')
```

Comment:
Curious: Why pick the least downloaded versions of the package?  Why not include the latest?